pkg/cmd/importcmd/import.go

```diff
@@ -407,13 +407,25 @@ func (o *ImportOptions) Run() error {
 	}
 	*/
 
-	gitOpsRepo, err := IsGitOpsRepositoryWithPipeline(o.Dir)
+	// lets disable the build pack if we have a jenkins-x.yml or a .lighthouse/*/triggers.yaml file
+	jxProjectFile := filepath.Join(o.Dir, config.ProjectConfigFileName)
```

Comment:
I guess [the PR](https://github.com/jenkins-x/jx-project/pull/86) can be closed then ^^

Reply:
thanks for helping though! :)